openapi.yml

```diff
@@ -0,0 +1,506 @@
+  /workflow_archive:
+    get:
+      summary: Retrieve archived workflows
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: workflow
+          in: query
+          required: false
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'
+  /workflow_queue:
+    get:
+      summary: Retrieve the workflow queue
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: completed
+          in: query
+          required: false
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+        - name: waiting
+          in: query
+          required: false
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+  /workflow_queue/lane_ids:
+    get:
+      summary: Retrieve the workflow queue based on the provided lane_id
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: step
+          in: query
+          required: true
+          example: dor:etdSubmitWF:catalog-status
+          schema:
+            type: string
+  /workflow_queue/all_queued:
+    get:
+      summary: Retrieve the workflow queue for all queues
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: limit
+          in: query
+          required: false
+          example: '10'
+          schema:
+            type: string
+components:
+  schemas:
+    Druid:
+      description: Digital Repository Unique Identifier (bare or prefixed)
+      type: string
+      pattern: '^(druid:)?[b-df-hjkmnp-tv-z]{2}[0-9]{3}[b-df-hjkmnp-tv-z]{2}[0-9]{4}$'
+      example: 'druid:bc123df4567'
+    Process:
+      description: Process name
+      type: string
+      enum: [accessioning-initiate, approve, check-marc, checksum-compute, complete-ingest, content-metadata,
+             content-metadata-create, descriptive-metadata, end-accession, exif-collect, jp2-create,
+             preservation-ingest-initiated, provenance-metadata, publish, publish-complete, reader-approval,
+             registrar-approval, remediate-object, reset-workspace, rights-metadata, sdr-ingest-received, sdr-ingest-transfer,
+             shelve, shelve-complete, start-accession, start-assembly, submit, technical-metadata, transfer-object, 
+             update-catalog, update-moab, validate-bag, verify-apo]
+    Repository:
```

Comment:
Is this the arg we're trying to phase out? If so, mark as deprecated in the spec?

Reply:
I can do that.

Reply:
Done.